libraries/AP_Camera/AP_Camera.h

```diff
@@ -115,6 +115,9 @@ class AP_Camera {
     void take_picture();
     void take_picture(uint8_t instance);
 
+    // take multiple pictures
```

Comment:
We should add a comment above this specifying that setting total_num = -1 means it will take pictures forever.